lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -87,6 +90,8 @@
 
   &.-centered {
     text-align: center;
+    width: 75%;
```

Comment:
use this instead

```
max-width: 75%;  // maybe 80%?
margin: 0 auto;
```

This will work alright on the regular feature card and wider cards as well. Don't do a fixed just `width` make it `max-width` so it's a bit more flexible.